### DISCUSSION TITLE
Pseudo tiling and master layout

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [hyprwm/Hyprland#1316] [General]
Is it just me or pseudotiling works only for dwindle layout, and so the pseudo dispatcher should be moved [from the list of general dispatchers](https://wiki.hyprland.org/Configuring/Dispatchers/#list-of-dispatchers) to the list of [dwindle-specific dispatchers](https://wiki.hyprland.org/Configuring/Dwindle-Layout/#bind-dispatchers)?

**Comment by vaxerski** [comment]
pseudo only dwindel yes
